### PR TITLE
fixed typo in first line

### PR DESCRIPTION
**Pull request by mas-designs**
There was becuase instead of because!

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by b3u** [comment]
LGTM

File: guide/english/jquery/jquery-click-method/index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ title: Click Method
 # Click Method
 
 
-The jQuery Click method triggers an function when an element is clicked. The function is known as a "handler" becuase it handles the click event. Functions can
+The jQuery Click method triggers an function when an element is clicked. The function is known as a "handler" because it handles the click event. Functions can
```
Comment (huyenltnguyen) [inline guide/english/jquery/jquery-click-method/index.md]: Hi @mas-designs, thank you for the PR. Your change looks good to me. However, there is another typo/grammar error in the first sentence:
```
The jQuery Click method triggers an function...
```
should be
```
The jQuery Click method triggers a function...
```

Would you mind fixing it as well?

**Comment by mas-designs** [comment]
@huyenltnguyen thank you for mentioning it! Totally missed this, I've updated the file, please let me know if there is anything else I should do!

**Comment by huyenltnguyen** [comment]
@mas-designs Thank you for the additional change! Your PR looks good to me. Unfortunately, it is not ready to merge at the moment due to the Travis CI build failure. This is totally not your fault. I think there was a recent change to the repo that caused it, and many PRs have been affected as well.

I will check with other moderators and try to get your PR merged ASAP.

**Review by huyenltnguyen — APPROVED** [review]

**Comment by huyenltnguyen** [comment]
Awesome! We have those green ticks back! I'm merging it now.

@mas-designs Thank you for the PR and congratulations on your first contribution to this repo ✨ 🎉 
Happy coding!

**Pull request merged**